run-sandbox.sh

```diff
@@ -41,10 +41,26 @@ if [ ! -f daml-on-sql-1.10.0.jar ]; then
    wget https://github.com/digital-asset/daml/releases/download/v1.10.0/daml-on-sql-1.10.0.jar
 fi
 
-java -jar daml-on-sql-1.10.0.jar \
+OCSP_STAPLE="TRUE"
+if [ "$OCSP_STAPLE" == "TRUE" ] ; then
+   export JDK_JAVA_OPTIONS="-javaagent:$ROOTDIR/jSSLKeyLog.jar=server.log -Djava.security.properties=$ROOTDIR/java.security -Dcom.sun.net.ssl.checkRevocation=true -Djdk.tls.client.enableStatusRequestExtension=true -Djdk.tls.server.enableStatusRequestExtension=true -Djava.security.debug='certpath ocsp' -Djavax.net.debug='ssl:handshake,verbose,respmgr'"
+   wget https://github.com/jsslkeylog/jsslkeylog/releases/download/v1.3.0/jSSLKeyLog-1.3.zip -O jSSLKeyLog-1.3.zip
+   unzip -o jSSLKeyLog-1.3.zip jSSLKeyLog.jar
+   JAR="daml-on-sql-binary_deploy.jar"
+   if [ ! -f $JAR ] ; then
+      echo "WARNING: OCSP depends on a custom build of daml-on-sql to use JDK SSLProvider"
+      exit 1
+   fi
```

Comment:
Let’s please remove this. Nobody reading this can (due to the lack of instructions) or should (due to the lack of testing & support) be using tghis.